appjs/views/edit_project.js

```diff
@@ -22,6 +22,74 @@ function isVisible(control) {
   return control.type !== 'hidden' && $(control.domEl).is(':visible');
 }
 
+var BaseModalView = Backbone.View.extend({
```

Comment:
Should rename to `ProjectSaveModal` or something since it's not a generic modal thing.